styles.css

```diff
@@ -0,0 +1,745 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+}
+
+.nav-icon-box {
+  display: flex;
+  background-color: #272a31;
+  width: 100%;
+  height: 30px;
+  margin: 0;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.nav-icon-box ul {
+  list-style-type: none;
+  margin: 0 10%;
+}
+
+.close-button {
+  display: none;
+}
+
+.nav-icon-box li {
+  display: inline;
+  margin: 0 10px;
+}
+
+.nav-icon-box img {
+  width: 16px;
+  height: 16px;
+}
+
+.nav-icon-box a {
+  font-family: lato, sans-serif;
+  font-weight: bold;
+  font-size: 12px;
+  text-decoration: none;
+  color: #d3d3d3;
```

Comment:
- [ ] [OPTIONAL]:
- It would be better if we use CSS variables here so we can save the custom colors used all over the CSS file instead of repeating the color code.